### PR TITLE
Expose needle position and currently dragging index

### PR DESCRIPTION
**Pull request by ben-styling**
Hi here, 

Firstly thank you for this library!

This PR includes 2 changes. 

1. `itemDragged` is exposed to the renderItem function
2. `needle` is exposed to the renderList function as a getter prop `getNeedle`

This change allows for some more flexible styling based on the "needle" position.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,6 +193,14 @@ container?: Element;
 
 Provide custom DOM element where moved item will be rendered.
 
+## getNeedle
+
+```ts
+getNeedle?: (needle: string) => void;
```
Comment (tajo) [inline README.md:199]: needle is number?

**Comment by tajo** [comment]
Can you add an example that uses these new APIs in a realistic case?

Side note: I need to fix CI tests / move them over to Github actions.

File: README.md (hunk 1 above)
Comment (ben-styling) [reply README.md]: Good catch, fixed :)

**Comment by tajo** [comment]
If you want to rebase/fix this, this would be a still good feature to have I think.

**Pull request closed without merging**